### PR TITLE
Add request timeout to client API and update package versions

### PR DESCRIPTION
**Pull request by martin-fleck-at**

### CODE REVIEW

File: node/common/src/client.ts [hunk 1]
```diff
@@ -107,10 +109,10 @@ export namespace TheiaCloud {
     }
   }
 
-  export async function launch(request: LaunchRequest, retries = 0): Promise<void> {
+  export async function launch(request: LaunchRequest, retries = 0, timeout?: number): Promise<void> {
     const launchRequest = { kind: LaunchRequest.KIND, ...request };
     try {
-      const response = await rootApi(request.serviceUrl).servicePost({ launchRequest }, createConfig());
+      const response = await rootApi(request.serviceUrl).servicePost({ launchRequest }, createConfig(timeout || request.timeout));
```
Comment (jfaltermeier) [inline node/common/src/client.ts:115]: The `request.timeout` is used by the Kubernetes Client. When the timeout is reached we set an error message on the session. I don't think this should be used here.

**Review by jfaltermeier — COMMENTED** [review]
Thank you for the PR. Looks good overall. 
I have one small comment however.

File: node/common/src/client.ts (hunk 1 above)
Comment (martin-fleck-at) [reply node/common/src/client.ts:115]: Fixed.

**Comment by martin-fleck-at** [comment]
@jfaltermeier Thank you for the review, I fixed it in a follow-up commit.

**Review by jfaltermeier — APPROVED** [review]
Thanks, LGTM!
The npm-packages are not yet released, so at the moment building the activity tracker sample fails.

**Pull request merged**